### PR TITLE
Update nodejs modules

### PR DESCRIPTION
**Pull request by tigerfoot**
Update karma from 0.8.5 to 0.12
Update less from 1.4.0 to 2.5
Update ng-annotate from 0.13.0 to 0.15.4

### CODE REVIEW

**Comment by oterral** [comment]
I'll review/test this PR today

File: package.json [hunk 1]
```diff
@@ -10,9 +10,12 @@
...
   "readmeFileName": "README.md",
   "devDependencies": {
-    "less": "1.4.0",
-    "karma": "0.8.5",
-    "ng-annotate": "0.13.0",
+    "less": "~2.5.0",
+    "less-plugin-clean-css": "~1.5.0",
+    "karma": "~0.12.0",
+    "karma-mocha": "0.1.10",
+    "karma-phantomjs-launcher": "0.1.4",
```
Comment (oterral) [inline package.json]: Why do we need  `karma-mocha' and `karma-phantomjs-launcher` now?

Comment (oterral) [inline package.json]: Forget this,  I've read the karma doc

**Comment by oterral** [comment]
This one lgtm , can you just keep the 2 spaces indentation in `test/karma-conf.mako.js` . 

Thx  for this.

**Comment by tigerfoot** [comment]
On hollidays until July 8th we will take care of it at that time.


À mer. juin 24 14:54:42 2015 GMT+0300, oterral a écrit :

**Comment by oterral** [comment]
@tigerfoot back to holidays? can you fix this , I would like to merge it.

**Comment by tigerfoot** [comment]
@oterral Now should be fixed, thanks for review

**Comment by oterral** [comment]
Thx @tigerfoot

**Pull request merged**